### DISCUSSION TITLE
Incorrect symbolic path of cache file when config cache path as relative

### DISCUSSION THREAD

**Discussion by ekko3add7** [treeverse/dvc#6876] [Help]
I had configured the cache dir as the relative path and type as symbolic.
Why are the file paths combined with an absolute path and relative path? 

`cat .dvc/config`

> [cache]
> dir = ../../dvc-test
> type = "symlink,copy"

`ls -l aaa`
> 1.txt -> /home/ekko.chang/dev/test/data-test/.dvc/../../dvc-test/d4/1d8cd98f00b204e9800998ecf8427e
> 2.txt -> /home/ekko.chang/dev/test/data-test/.dvc/../../dvc-test/d4/1d8cd98f00b204e9800998ecf8427e

**Comment by efiop** [comment]
Definitely a bug on our side. Need to take a closer look. In the mean time, this works fine for you, right? Just visually looks odd?

**Reply by ekko3add7** [reply]
Hello @efiop 
It is a little bit problem when I bind these data files into the container because the real data path is mounted on the **host** from a remote NAS server.
Of course, it could be solved with another workaround, just confirm the problem with you.

**Reply by efiop** [reply]
@ekko3add7 With bind mounts, I guess the problem here is not so much `..` in the path, but more the fact. that we use use symlinks to an absolute path, right?

**Reply by ekko3add7** [reply]
@efiop yes, as I expect the symlinks as relative path should be like this:
`1.txt -> ../../dvc-test/d4/1d8cd98f00b204e9800998ecf8427e`

**Comment by karajan1001** [comment]
Create an issue for it?
